src/coinbase/historical_balance.ts

```diff
@@ -35,7 +35,7 @@ export class HistoricalBalance {
   public static fromModel(model: HistoricalBalanceModel): HistoricalBalance {
     const asset = Asset.fromModel(model.asset);
     return new HistoricalBalance(
-      asset.fromAtomicAmount(new Decimal(model.amount)),
+      model.amount != "" ? asset.fromAtomicAmount(new Decimal(model.amount)) : new Decimal(0),
```

Comment:
I am not sure if I understand why are we touching the historical balance file ? The only 2 files related to our team's rewards related work are `staking_balance.ts` and `staking_reward.ts`